### ISSUE TITLE
Final gamepad controls and controller map 

### ISSUE THREAD

**Issue by ReflextionsDev** [ReflextionsDev/Zombiehood#30]
- [ ] Review all controls are supported on gamepad (including laser, weapon drop, etc)
- [ ] Create in game controller layout on menu
- [ ] Dynamically go to correct "options" layout depending on gamepad detection

**Comment by ReflextionsDev** [comment]
For reference:

![image](https://github.com/user-attachments/assets/991661c0-47d3-436c-9f45-4b86c1d2311e)

Need to find open source game image

**Comment by ReflextionsDev** [comment]
Map is in, need to finalize d-pad and extra controls

**Comment by Everade** [comment]
> Map is in, need to finalize d-pad and extra controls

Weapon drop (d-pad down) has now added with https://github.com/ReflextionsDev/Zombiehood/pull/89. Currently the controls are according to the ingame gamepad interface image. So should be finished?!

**Comment by ReflextionsDev** [comment]
I forget, do we have melee? Also does hold to scrap work?

**Comment by Everade** [comment]
Yes meele works. Hold to scrap, not that i'm aware off. What's that supposed to do? D-Pad down just drops the weapon immediately, like on keyboard.

**Comment by ReflextionsDev** [comment]
Tested it locally, looks good! Yes you can actually hold interact to sell items at any time. It's a pretty hidden feature lol. Is working on gamepad though.

**Comment by Everade** [comment]
Just tested it, so you can scrap items that are currently on the floor.
I suppose we should add that info to the gamepad UI and we're done with this issue?
